index.html

```diff
@@ -24,6 +24,12 @@
 	<div class="nav">
 		<ul>
 			<li><a href="animals_and_tips.html"> Link 1</a>
+			
+				<ul>

```

Comment:
These page links were convient for faster acess to infromation as this page is very long